File: Hurb/Hurb/Model/Meta.swift
//
//  Meta.swift
//  Hurb
//
//  Created by Alexandre Papanis on 08/08/19.
//  Copyright © 2019 Papanis. All rights reserved.
//

import Foundation

struct Meta: Codable {
    let count: Int?
    let offset: Int?
    let query: String?
    let warning: String?
    let countWithAvailabilityInPage: Int?
    let countHotel: Int?
    let countPackage: Int?
    let countTicket: Int?
    let countBustrip: Int?
    let countDisney: Int?
}

File: HurpApp/app/src/main/java/app/recrutamento/android/challengealpha/model/hotel/Meta.kt
package app.recrutamento.android.challengealpha.model.hotel

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Meta {

    @SerializedName("count")
    @Expose
    var count: Int? = null
    @SerializedName("offset")
    @Expose
    var offset: Int? = null
    @SerializedName("query")
    @Expose
    var query: String? = null
    @SerializedName("warning")
    @Expose
    var warning: String? = null
    @SerializedName("countWithAvailabilityInPage")
    @Expose
    var countWithAvailabilityInPage: Int? = null
    @SerializedName("countHotel")
    @Expose
    var countHotel: Int? = null
    @SerializedName("countPackage")
    @Expose
    var countPackage: Int? = null
    @SerializedName("countTicket")
    @Expose
    var countTicket: Int? = null
    @SerializedName("countBustrip")
    @Expose
    var countBustrip: Int? = null
    @SerializedName("countDisney")
    @Expose
    var countDisney: Int? = null

}
